packages/btcindexer/src/btcindexer.ts

```diff
@@ -363,7 +390,13 @@ export class Indexer implements Storage {
 					});
 				}
 			} catch (e) {
-				console.error(`Minting: ERROR preparing TX ${txId}:`, e);
+				const error =
+					e instanceof Error ? { name: e.name, msg: e.message, stack: e.stack } : e;
```

Comment:
mapping an error is repeated, could you create a function for it in a new module, eg `errutils.ts`?

Reply:
done